src/transactions/feed/TransactionFeedV2.tsx

```diff
@@ -409,7 +419,7 @@ export default function TransactionFeedV2() {
       // scroll actions for loading additional pages and the initial wallet load
       // if no cached transactions exist.
       if (('hasAfterCursor' in error && error.hasAfterCursor) || feedFirstPage.length === 0) {
-        setShowError(true)
+        setStatus('error')
       }
```

Comment:
@kathaypacific what do you think of moving this to a derived state? It seems like the only scenario when we use this `error` status is the same as it was in the previous revision with `showError` variable. 
```typescript
const showRefetchError = useMemo(() =>  {
  if (error === undefined) return false;
  return ( 'hasAfterCursor' in error && error.hasAfterCursor) || feedFirstPage.length === 0
}, [error, feedFirstPage])
```

Reply:
the main reason i wanted to have this enum value for `status` rather than have independent states for error and loading is to prevent both states from being true at the same time (and to avoid needing to put in extra logic to sync these 2 statuses) - the behaviour i want is when there is a fetch error, the user pulls to refresh, the loader should be displayed and the error is hidden (since at this point, the error is outdated + it's more clear that there is a new error if the error is first dismissed during loading). the status enum is this is a pattern we use in a few places, like the positions slice.